src/libp2p/connectionsManager.ts

```diff
@@ -289,6 +289,10 @@ export class ConnectionsManager {
     if (response.status !== 200) {
       switch (response.status) {
         case 403:
+          this.emitCertificateRegistrationError('Username already taken.')
+          break
+        default:
+          this.emitCertificateRegistrationError('Registering username failed.')
```

Comment:
maybe join this in switch ?
```
    switch (response.status) {
        case 200:
           break
        case 403:
          this.emitCertificateRegistrationError('Username already taken.')
          break
        default:
          this.emitCertificateRegistrationError('Registering username failed.')
          break
```
this way you don't have this wrapping `if (response.status) ..`